templates/settings.yml

```diff
@@ -0,0 +1,104 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/
+repository:
+  homepage: "https://developer.lightbend.com/start/?group=play"
+  topics: playframework, example, example-project, sample, sample-app, jvm, webapp
+  private: false
+  has_issues: true
+  # We don't need projects in sample projects
+  has_projects: false
+  # We don't need wiki in sample projects
+  has_wiki: false
+  has_downloads: true
+  default_branch: 2.7.x
+  allow_squash_merge: true
+  allow_merge_commit: false
+  allow_rebase_merge: false
+
+teams:
+  - name: core
+    permission: admin
+  - name: integrators
+    permission: write
```

Comment:
I think we need this, because the CLA validator is in write-bots (below).

Reply:
Yes.

Reply:
Oops, I meant I _don't_ think we need this, because the CLA validator is in write-bots